### PR TITLE
autoprefixer?

### PR DESCRIPTION
**Pull request by van-nguyen**
Can we have autoprefixer in the build process or a way to extend it in our project? Thanks.

### CODE REVIEW

**Comment by van-nguyen** [comment]
Cool, have converted this issue into a pull request. Have added as a Less plugin instead to reduce dependencies and overhead.

**Comment by van-nguyen** [comment]
The browser support is somewhat opinionated abit, perhaps we can have it as a config somewhere?

**Comment by martijndeh** [comment]
This looks great.

The browser support is indeed a bit opinionated and is nicer as a config. There is an existing settings system in the app.

In user-land, you can either set settings when constructing the app:
```js
var app = require('fire')('nodeonfire.org', {
    modules: ['angular-route', 'angular-moment'],
    NODE_ENV: process.env.NODE_ENV,
    less: {
        plugins: [new LessPluginAutoPrefix({browsers: ['> 5%', 'last 2 versions']})]
    }
});
```

Or use `App#settings`:
```js
app.settings('less', {
    plugins: [new LessPluginAutoPrefix(...)]
});
```

So it would be nice if we can pass the plugins directly to the less build (and by default use the LessPluginAutoPrefix)? Optionally also add a setting to configure the browser support?

What do you think?

**Comment by van-nguyen** [comment]
Cool, thanks for the info, I have made it extra flexible to fit all use cases.

* defaults to using autoprefixer with a sane value such as > 5% and last 2 versions
* ability to turn off autoprefixer by setting `autoprefixer = false`:
* ability to set browser support for autoprefixer without including the plugin in their project
* pass additional plugins to the build process

File: lib/modules/build-system/index.js [hunk 1]
```diff
@@ -199,6 +200,19 @@ function BuildSystem(stageMethods, $q) {
             buildPath = path.join(basePath, '.fire', '.build', 'public', 'styles');
         }
 
+        var lessConfig = app.get('less');
```
Comment (martijndeh) [inline lib/modules/build-system/index.js]: This should become `app.settings` actually.

`app.get` also exists, but is used for express-like middleware.

**Comment by martijndeh** [comment]
The rest looks great.

**Comment by van-nguyen** [comment]
Oops don't know why I did that, thanks for pointing it out. Fixed.

**Pull request merged**